a11y/FrontendAccessibilitySpec.scala

```diff
@@ -1,4 +1,4 @@
-import models.{CheckMode, Mode, NormalMode}
+import models.{Mode, NormalMode}
```

Comment:
Do we want accessibility for check mode, too? Maybe not a question for this ticket.

Reply:
Was wondering that as I removed the import! I think right now the pages are so similar in each mode its not a big concern, but maybe something for down the line

Reply:
Cool!